docs/podman-containers-cleanup.1.md

```diff
@@ -0,0 +1,38 @@
+% podman-container-cleanup "1"
+
+## NAME
+podman\-container\-cleanup - Cleanup Container storage and networks
+
+## SYNOPSIS
+**podman** **container-cleanup** [*options* [...]] container
+
+## DESCRIPTION
+`podman container cleanup` cleans up exited containers by removing all mountpoints and network configuration from the host.  The container name or ID can be used.  The cleanup command does not remove the containers.  Running containers will not be cleaned up.
+Some times containers mount points and network stacks can remain if the podman command was killed or the container ran in daemon mode.  This command is automatically executed when you run containers in daemon mode by the conmon process when the container exits.
+
+## OPTIONS
+
+**--all, a**
+
+Cleanup all containers.
+
+**--latest, -l**
+Instead of providing the container name or ID, use the last created container. If you use methods other than Podman
+to run containers such as CRI-O, the last started container could be from either of those methods.
+## EXAMPLE
+
+podman container cleanup mywebserver
+
+podman container cleanup mywebserver myflaskserver 860a4b23
+
+podman container cleanup -f 860a4b23
+
+podman container-cleanup -a
+
+podman container cleanup --latest
```

Comment:
I think you want fencing backticks or four-space indents for all of these examples.  And also possibly English explaining the point you're illustrating with each.

Reply:
fixed